Plugins/GenerateManual/GenerateManualPluginError.swift

```diff
@@ -10,7 +10,11 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
+#if swift(>=6)
+@preconcurrency import PackagePlugin
```

Comment:
Hmm it feels like we should file a bug on SwiftPM for this

Reply:
I can't reproduce the warnings that led me to add this anymore 😭